resources/admin/classes/common/_BKP_Dashboard.php

```diff
@@ -0,0 +1,264 @@
+        $acquisitionType = new AcquisitionType();
+        echo '<select name="acquisitionTypeID" id="acquisitionTypeID" style="width:150px;">';
+        echo "<option value=''>All</option>";
+        foreach($acquisitionType->allAsArray() as $display) {
+            if ($display['acquisitionTypeID'] == $current) {
+                echo "<option value='" . $display['acquisitionTypeID'] . "' selected>" . $display['shortName'] . "</option>";
+            } else {
+                echo "<option value='" . $display['acquisitionTypeID'] . "'>" . $display['shortName'] . "</option>";
+            }
+        }
+        echo '</select>';
+    }
+
+    function getOrderTypesAsDropdown($currentID = null) {
+        $display = array();
+        $orderType = new OrderType();
+        echo '<select name="orderTypeID" id="orderTypeID" style="width:150px;">';
+        echo "<option value=''>All</option>";
+        foreach($orderType->getAllOrderType() as $display) {
+            if ($display['orderTypeID'] == $current) {
+                echo "<option value='" . $display['orderTypeID'] . "' selected>" . $display['shortName'] . "</option>";
+            } else {
+                echo "<option value='" . $display['orderTypeID'] . "'>" . $display['shortName'] . "</option>";
+            }
+        }
+        echo '</select>';
+    }
+
+    function getSubjectsAsDropdown($currentID = null) {
+        $generalSubject = new GeneralSubject();
+        $generalSubjectArray = $generalSubject->allAsArray();
+
+        $detailedSubject = new DetailedSubject();
+        $detailedSubjectArray = $detailedSubject->allAsArray();
+
+        echo '<select name="subjectID" id="subjectID" style="width:150px;">';
+        echo "<option value=''>All</option>";
+        foreach($generalSubjectArray as $ug) {
+            $generalSubject = new GeneralSubject(new NamedArguments(array('primaryKey' => $ug['generalSubjectID'])));
+            echo "<option value='" . $ug['generalSubjectID'] . "'>" . $ug['shortName'] . "</option>";
+            foreach ($generalSubject->getDetailedSubjects() as $ds){
+                echo "<option value='d" . $ds->detailedSubjectID . "'> -- " . $ds->shortName . "</option>";
+            }
+        }
+        echo '</select>';
+    }
+
+    function getCostDetailsAsDropdown($currentID = null) {
+        $display = array();
+        $costDetails = new CostDetails();
+        echo '<select name="costDetailsID" id="costDetailsID" style="width:150px;">';
+        echo "<option value=''>All</option>";
+        foreach($costDetails->allAsArray() as $display) {
+            if ($display['costDetailsID'] == $current) {
+                echo "<option value='" . $display['costDetailsID'] . "' selected>" . $display['shortName'] . "</option>";
+            } else {
+                echo "<option value='" . $display['costDetailsID'] . "'>" . $display['shortName'] . "</option>";
+            }
+        }
+        echo '</select>';
+
+    }
+
+
+
+}
+?>
```

Comment:
@aniqaamir did you intend to include this file with your pull request? It looks like a new file but it doesn't appear to be referenced by any of your code changes.